roles/lib_utils/callback_plugins/aa_version_requirement.py

```diff
@@ -29,13 +31,15 @@ def display(*args, **kwargs):
 
 
 # Set to minimum required Ansible version
-REQUIRED_VERSION = '2.5.7'
+REQUIRED_VERSION = version.StrictVersion('2.5.7')
 DESCRIPTION = "Supported versions: %s or newer" % REQUIRED_VERSION
 
 
-def version_requirement(version):
+def version_requirement(ver):
     """Test for minimum required version"""
-    return version >= REQUIRED_VERSION
+    if not isinstance(ver, version.StrictVersion)
```

Comment:
`/roles/lib_utils/callback_plugins/aa_version_requirement.py:40:49: E999 SyntaxError: invalid syntax`

Reply:
oops! Sorry about that